src/main.rs

```diff
@@ -0,0 +1,32 @@
+fn main() {
+    println!("Bem vindo ao TODO List");
+    loop{
+        if !create_todo(){
+            break
+        }   
+    }
+}
+
+fn input() -> String {
+    let mut buf = String::new();
+    std::io::stdin().read_line(&mut buf).unwrap();
+    buf.trim().to_string()
+ }
+
+fn create_todo() -> bool{
+    println!("Gostaria de criar um novo TODO? (s/n)");
+    let anwser_todo = input();
+
+    if anwser_todo == "s" || anwser_todo == "S"{
+        println!("Qual TODO gostaria de criar?");
+        let todo_user: String = input();
```

Comment:
```suggestion
        let todo_user = input();
```
Aqui o compilador consegue inferir o tipo da variável a partir do tipo do retorno da função `input`, que é `String`.